scripts/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh -e
+
+if [ -z "${SOURCE_DIR}" ]; then
+  SOURCE_DIR="$(cd "$(dirname $0)" && git rev-parse --show-toplevel)"
+else
+  SOURCE_DIR="$(cd "${SOURCE_DIR}" && pwd)"
+fi
+
+# Link user presets for this system if they don't exist
+if [ ! -e "CMakeUserPresets.json" ]; then
+  _USER_PRESETS="scripts/cmake-presets/${HOSTNAME%%.*}.json"
+  if [ -f "${_USER_PRESETS}" ]; then
+    ln -s "${_USER_PRESETS}" "CMakeUserPresets.json"
+  fi
+fi
+
+cd "${SOURCE_DIR}"
```

Comment:
Shouldn't this be before line 10? (and/or is it already assume to be the current directory?)